test/e2e/storage/sanity.go

```diff
@@ -294,6 +294,22 @@ var _ = Describe("sanity", func() {
 			}, "3m", "5s").Should(Equal(capacity.AvailableCapacity), "total capacity after controller restart")
 		})
 
+		It("should return right capacity", func() {
+			resp, err := cc.GetCapacity(context.Background(), &csi.GetCapacityRequest{
+				AccessibleTopology: &csi.Topology{
+					Segments: map[string]string{
+						"pmem-csi.intel.com/node": nodeID,
+					},
+				},
+			})
+			Expect(err).NotTo(HaveOccurred())
+			cap := resp.AvailableCapacity
+			resp, err = ncc.GetCapacity(context.Background(), &csi.GetCapacityRequest{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.AvailableCapacity).To(Equal(cap), "capacity mismatch")
```

Comment:
If all nodes have the same amount of free storage, then this test cannot verify that the call with topology information really ends up querying the right node.

Can you make it a bit more strict?
- check and compare results for all nodes
- create a volume on one node
- check and compare all nodes again

Reply:
> If all nodes have the same amount of free storage, then this test cannot verify that the call with topology information really ends up querying the right node.
> 
> Can you make it a bit more strict?
> 
> * check and compare results for all nodes
> * create a volume on one node
> * check and compare all nodes again

This might not work. Capacity is the maximum available extent in a region/vgs on the node. So, creating a volume might not change in the Node capacity, if the volume is created in the smaller region/vgs.

Reply:
Then repeat creating volumes until there is a change in capacity.